core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -592,6 +592,19 @@ object Parser extends ParserInstances {
       softProduct01(parser, void1(that)).map(_._1)
   }
 
+  /** Methods with complex variance type signatures due to covariance.
+    */
+  implicit final class Parser1Methods[A](private val self: Parser1[A]) extends AnyVal {
+    def repAs[B](implicit acc: Accumulator[A, B]): Parser[B] =
+      Parser.repAs(self)(acc)
+
+    def rep1As[B](implicit acc: Accumulator1[A, B]): Parser[B] =
```

Comment:
Bikeshedding but maybe `repAs1` is better since the 1 is more obvious as a suffix (and that's where we put it in other cases).

Reply:
yeah, I think your suggestion is right.